thegame/engine/camera.py

```diff
@@ -0,0 +1,98 @@
+""" A camera class used by base_game to handle showing only portions of each
+    map at a time."""
+import logging
+import pprint
+
+from .map import Map
+
+
+class Camera:
+    """ A Camera class for dealing with the game camera."""
+
+    def __init__(self, camera_width, camera_height, camera_x, camera_y):
+        """ In order to have a center of the screen, camera_width and camera_height
+            act as if they are the next highest odd number (presuming they're even.)
+            If its odd, the value doesn't change."""
+        self.camera_width = camera_width
+        self.camera_height = camera_height
+        self.camera_x = camera_x
+        self.camera_y = camera_y
+
+    def get_camera_fov(self, game_map: Map):
+        #  x x x x x
+        #      ^
+        # camera_x = 2
+        # camera_width = 3
+        # 1 - (3 // 2) = 2 - 1 = 1
+        # 1 + (3 // 2) = 2 + 1 = 3
+        #
+        leftmost_sprite = self.camera_x - (self.camera_width // 2)
+        rightmost_sprite = self.camera_x + (self.camera_width // 2)
+
+        topmost_sprite = self.camera_y - (self.camera_height // 2)
+        bottommost_sprite = self.camera_y + (self.camera_height // 2)
+
+        # TODO: There may be a better way to do this.
+        # Current functionality:
```

Comment:
Add the None functionality to this break down.

Reply:
Will do.

Reply:
Addressed with https://github.com/Ross-Alexandra/TheGame/pull/16/commits/1711bd575b771a88304b64123375c7bdb693ac66